File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var sumOfUnique = function(nums) {
  const set = new Set(nums)
  nums.forEach((n, i) => nums.indexOf(n) != i ? set.delete(n) : 0)
  if (set.size === 0) return 0
  return [...set].reduce((a, b) => a + b)
};

File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.go
func sumOfUnique(nums []int) int {
  dict := make(map[int]int, len(nums))
  for _, v := range nums {
    dict[v]++
  }
  ans := 0
  for k, v := range dict {
    if v == 1 {
      ans += k
    }
  }
  return ans
}